src/platform/mbed/net/src/net_if.cpp

```diff
@@ -405,13 +441,175 @@ char * mbed_inet_ntop(sa_family_t family, const void * src, char * dst, size_t s
     case AF_INET6:
         return (inet_ntop6(src, dst, (size_t) size));
     default:
+        _set_errno(EAFNOSUPPORT);
         return (NULL);
     }
 }
 
+/* Return the value of CH as a hexademical digit, or -1 if it is a
+   different type of character.  */
+static int hex_digit_value(char ch)
+{
+    if ('0' <= ch && ch <= '9')
+        return ch - '0';
+    if ('a' <= ch && ch <= 'f')
+        return ch - 'a' + 10;
+    if ('A' <= ch && ch <= 'F')
+        return ch - 'A' + 10;
+    return -1;
+}
+
+static int inet_pton4(const char * src, void * dst)
```

Comment:
This is defined in several places in mbed-os: LWIP, nanostack, ... Can you create a ticket so we factorize it at the os level rather than in another library ?

Reply:
Do you mean to use inet_ntop and inet_pton functions implementation from mbed depending on what the stack is enabled?